assets/training/model_management/environments/foundation-model-inference/context/requirements.txt

```diff
@@ -1,17 +1,22 @@
-pandas==2.0.3
-numpy==1.25.0
-text-generation==0.6.0
-azureml-inference-server-http==0.8.4
-azureml-core==1.52.0
-azureml-contrib-services==1.52.0
+git+https://github.com/microsoft/DeepSpeed.git
+git+https://github.com/srsaggam/DeepSpeed-MII.git@srsaggam/hf_conversational3
```

Comment:
What is the plan to merge this to the official repo?

Reply:
We will be using the patch version that is pending release from Deepspeed MII team